src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdates.groovy

```diff
@@ -78,9 +79,12 @@ class DependencyUpdates {
     Map<Map<String, String>, String> downgradeVersions = toMap(versions.downgrade)
     Map<Map<String, String>, String> upgradeVersions = toMap(versions.upgrade)
 
+    // check for gradle versions
```

Comment:
nit: comments should be sentences https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html